package.json

```diff
@@ -42,8 +42,10 @@
     "event-kit": "^1.3.0",
     "fuzzaldrin-plus": "^0.3.1",
     "mkdirp": "^0.5.1",
+    "minimatch": "^3.0.4",
     "osenv": "^0.1.3",
-    "touch": "^1.0.0"
+    "touch": "^1.0.0",
+    "stringmap": "~0.2.2"
```

Comment:
Note: stringmap is already used a dependencies of dependencies, so this shouldn't increase load time